<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetCode</class>
 <widget class="QDialog" name="DialogSetCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get_base_image">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Base Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_bg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>NAME : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_name_change">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
